### ISSUE TITLE
Double indication of values used from opens_codelens causing view scrolling off-by-one erro

### ISSUE THREAD

**Issue by canadaduane** [jaredly/reason-language-server#228]
When "Indicate what values have been used from an open" is checked in the settings (under `Reason_language_server: Opens_codelens` section of vs code settings) I get one line and one _extra_ line in my code showing the values that the `open` adds to a file. The extra line seems to result in a miscalculation of how many lines are in the file, which results in (an annoying) off-by-one error that scrolls the view unnecessarily by one line whenever I type a key.

![off-by-one](https://user-images.githubusercontent.com/129/51090695-fb1d7880-173c-11e9-861a-24e7bf85c22d.png)

In case it helps, this is the `Float` module that I reference in the `Utils` parent module above:

```
module Float = {
  let (+) = (+.);
  let (-) = (-.);
  let ( * ) = ( *. );
  let (/) = (/.);
  let (~-) = (~-.);
};
```

This is what the off-by-one error does when editing the file (37 sec. video):
https://www.dropbox.com/s/tsp22oawsqubyfq/OffByOne.mp4?dl=0

(Each keypress, after a few hundred milleseconds debounce, results in the view scrolling by one line).

**Comment by jaredly** [comment]
Yeah, this happens when you have an error in your file, so analysis is temporarily disabled until the error is resolved. From your screenshot, you have an error on line one, and so the codelenses won't be correct.
But yeah the issue where it jumps the screen around is super annoying, and super needs to be fixed.

**Comment by canadaduane** [comment]
This is slightly off topic, but the error on line 1 is "Unbound module Buckle" which seems odd to me. Probably my setup is wrong.

**Comment by donut** [comment]
This is really painful. Another video of the issue: https://www.rightthisminute.com/tools/jw/videos/atAB8f2x/play?force_hq=1

Running `bsb -clean` doesn't solve it for me. But restarting vscode every 15 minutes when this starts happening does let it work for a bit.

**Comment by donut** [comment]
It looks like VSCode's April 2019 update [addresses this](https://code.visualstudio.com/updates/v1_34#_stable-codelens) with "Stable CodeLens". I re-enabled code lens for reason-vscode and things are going smoothly so far.

**Comment by farukg** [comment]
@jaredly 
> Yeah, this happens when you have an error in your file, so analysis is temporarily disabled until the error is resolved. 

i was searching for an issue i have, which you just described in that quote, i think. Is it possible to keep on showing codelens even if there is an error? cause thats actually the most needed time and all lenses are just shutdown :( If you wish, i file a separate issue for this, cause its not really related to  this one.
